lib/systems/src/DrivebrainController.cpp

```diff
@@ -0,0 +1,39 @@
+#include "DrivebrainController.h"
+
+
+TorqueControllerOutput_s DrivebrainController::evaluate(const SharedCarState_s &state)
+{
+
+    auto sys_tick = state.systick;
+    auto db_input = state.db_data;     
+    
+    bool message_too_latent = (::abs((int)(sys_tick.millis - db_input.last_receive_time_millis)) > (int)_params.allowed_latency);
```

Comment:
latency should be measured by sending a timestamp from VCR to DB and having DB include that timestamp in its next command message. Time between messages is important too but we should be measuring both these things.